src/olympia/constants/categories.py

```diff
@@ -148,7 +148,9 @@ def _immutable(self, *args):
             'social-networking': StaticCategory(
                 id=148, name=_(u'Social Networking')),
             'sports-games': StaticCategory(id=146, name=_(u'Sports & Games')),
-            'user-interface': StaticCategory(id=152, name=_(u'User Interface'))
+            'user-interface': StaticCategory(
+                id=152, name=_(u'User Interface')),
+            'other': StaticCategory(id=153, name=_(u'Other'), weight=333)
```

Comment:
its weird we don't set `misc=true` on any of the StaticCategory declarations, but rely on the bit of code below that does it based on slug.  🤷‍♂️

Reply:
Yeah... I don't remember why we (probably I...) did it like this. It might be worth looking into fixing that.